Kitodo/src/main/resources/messages/messages_de.properties

```diff
@@ -336,6 +336,7 @@ formConfigurationMismatch=Unstimmige Formularkonfiguration\:
 from=von
 gallery=Galerie
 generate=Generieren
+generator={0}-Images erzeugen
```

Comment:
why is there a "-"
It should be "3 Images erzeugen" rather than "3-Images erzeugen"
And why is it "Image" in german and not "Bild"?

Reply:
Was changed to “Bilder”. The placeholder is replaced by the name of the folder. It is therefore called e.g. “MAX-Bilder erzeugen”.